src/components/ContextSwitcher/context-switcher-utils.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
-import { Divider, DrilldownMenu, MenuItem } from '@patternfly/react-core';
+import { Divider, DrilldownMenu, Flex, FlexItem, MenuItem } from '@patternfly/react-core';
+import LockOpenIcon from '@patternfly/react-icons/dist/js/icons/lock-open-icon';
 import { ContextMenuItem } from './ContextSwitcher';
 
+export enum ItemVisibility {
+  PRIVATE = 'private',
+  COMMUNITY = 'community',
+}
+
+export const WorkspaceLabels = {
+  HAS_DIRECT_ACCESS: 'internal.workspaces.konflux-ci.dev/has-direct-access',
+  IS_OWNER: 'internal.workspaces.konflux-ci.dev/is-owner',
+};
```

Comment:
These are specific to the workspace switcher. shouldn't be part of ContextSwitcher